aims/aim2.md

```diff
@@ -0,0 +1,23 @@
+# Aim 2
+
+**To evaluate the extent to which low-dimensional representations of single cell data learned by various methods can be used to decompose tissues in the context of autoimmune/rheumatic diseases.**
+
+## Background
+
+Human Cell Atlas' partnership with the Immunological Genome Project (immgenH) will result in single-cell gene expression-based immunocyte phenotyping at unprecedented resolution.
+A compendium comprised of bulk gene expression data from autoimmune/rheumatic diseases is exceptionally well-suited for the biologically-motivated evaluation (e.g., benchmarking) of these immunocyte data.
+The _objective of this aim_ is to build such a compendium and to evaluate methods, including VAEs and other methods developed through this initiative, for defining cell-type-specific expression signatures from single-cell data by measuring their performance in the context of decomposing bulk, whole-tissue autoimmune/rheumatic disease data.
+The quality of the cell-type-specific signatures (and by extension the methods) will be measured by the by using this information in deconvolution frameworks and in correctly identifying cell-type proportion and phenotype in the context of both simulation and real data experiments.
+
+## Approach
+
+We will generate simulated bulk datasets drawn from purified cell lineages, combining the purified cell expression data at different proportions.
+We will use single-cell data-derived cell-type-specific signatures from different methods (e.g., VAEs, other methods developed through this initiative) as input to methods for decomposing bulk data and evaluate performance (e.g., prediction of proportions). 
+Bulk transcriptomes of homogenous cell types perturbed with many stimuli, such as those proposed by Arjun Raj's group, are well-suited for this experiment and will allow us to specifically evaluate performance with respect to cell-type proportion estimates and cell-type phenotype when highly similar cell-type states are added at different proportions.
```

Comment:
Are we proposing to use this data to generate samples? or as a positive control to see that specific single cell signatures are precisely activated in homogenized bulk?